include/nbl/builtin/hlsl/cpp_compat/basic.h

```diff
@@ -57,6 +58,7 @@ namespace nbl::hlsl
 #define NBL_CONSTEXPR_OOL_MEMBER const
 #define NBL_CONSTEXPR_INLINE_OOL_MEMBER const
 #define NBL_IF_CONSTEXPR(...) if (__VA_ARGS__)
+#define NBL_ASSERT(...)
```

Comment:
wel aready have a shader assert somwhere, it does a vk::RawBufferStore to an invalid (0) BDA

Reply:
See its actually here:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/1e637c44437cc1d0175127cb86520232c975da82/include/nbl/builtin/hlsl/macros.h#L13